CHANGELOG.md

```diff
@@ -7,6 +7,8 @@ and this project's packages adheres to [Semantic Versioning](http://semver.org/s
 
 ## [Unreleased]
 
+## [4.6.0] - 2024-11-28
+
 ### Changed
 
 - Use Phase instead of Status field for Clusters and Nodepools.
```

Comment:
Would be nice to mention the command(s) affected here

Reply:
Good idea, I updated the changelog to add the affected commands